internal/output/output.go

```diff
@@ -0,0 +1,30 @@
+package output
+
+import (
+	"io"
+	"os"
+)
+
+var OriginalStderr *os.File
+
+func init() {
+	OriginalStderr = os.Stderr
+}
+
+func CaptureAllOutput(to io.Writer) error {
+	piper, pipew, err := os.Pipe()
+	if err != nil {
+		return err
+	}
+
+	os.Stdout = piper
+	os.Stderr = piper
+
+	go func() {
+		_, err := io.Copy(to, pipew)
+		if err != nil {
```

Comment:
nbd, but could we instead `_, _ = io.Copy(to, pipew)`?
(mostly just so that static analysis doesn't report an empty branch)